ui/apps/platform/cypress/integration/violations/Violations.helpers.js

```diff
@@ -8,7 +8,7 @@ import { visit } from '../../helpers/visit';
 export const alertsAlias = 'alerts';
 export const alertsCountAlias = 'alertscount';
 
-const routeMatcherMapForViolationsWithoutSearchOptions = {
```

Comment:
It seems these changes come from a merge and should not show up in this PR. Perhaps GitHub bug?

Reply:
Ah oh no, this is likely when I merged the upstream branch from GitHub after I merged in the branch with the fix for OOM errors, which isn't typical to my workflow. I'll fix this up and re-push.